### PR TITLE
Feature/readme cmaf

### PR DESCRIPTION
**Pull request by NoeliaBentancor**
## Description
Please describe the changes.
If it's a feature or enhancement please be as detailed as possible.
If it's a bug fix, please link the issue that it fixes or describe the bug in as much detail.

## Requirements Checklist
- [ ] Unit Tests updated or fixed
- [ ] Docs/guides updated

### CODE REVIEW

File: lib/src/cmaf/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  hamToDash,
+  hlsToHam,
+  dashToHam,
+  Presentation,
+  getTracksFromPresentation,
+  validateTracks,
+} from "@svta/common-media-library";
+
+// Validate tracks in a presentation
+const validateTracks = (presentation: Presentation) => {
```
Comment (matiasrb97) [inline lib/src/cmaf/README.md]: Let's keep the validation outside the readme until we improve the validation flow in the PoC

File: lib/src/cmaf/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Features 
+
+* Fundamental on-demand conversion between [HLS] and [DASH] manifests, covering transformations such as HLS to HAM, HAM to HLS, DASH to HAM, and HAM to DASH.
+* Elementary Media Presentation querying functionality.
+* Validation of tracks for enhanced integrity assurance.
+
+## Usage 
+
+Here's a sample demonstration illustrating how to utilize the features of the CMAF HAM module in TypeScript:
```
Comment (DuckyCB) [inline lib/src/cmaf/README.md]:
```suggestion
Here's a simple demonstration illustrating how to use the features of the CMAF HAM module in TypeScript:

```

File: lib/src/cmaf/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  validateTracks,
+} from "@svta/common-media-library";
+
+
+// Parse from DASH format
+const parseFromDash = (input: string) => {
+  const hamObj = mpdToHam(input); // Convert input to HAM object
+  const hlsManifest = hamToHls(hamObj); // Convert HAM object to HLS manifest
+  const dashManifest = hamToDash(hamObj); // Convert HAM object to DASH manifest
+};
```
Comment (DuckyCB) [inline lib/src/cmaf/README.md]:
```suggestion
const originalDashManifest = fetchManifest();

const hamObject = dashToHam(originalDashManifest); // Converts Dash manifest to HAM Object

const hlsManifest = hamToHls(hamObject); // Convert HAM object to HLS manifest
const dashManifest = hamToDash(hamObject); // Convert HAM object to DASH manifest

...
```

File: lib/src/cmaf/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+// Parse from HLS format
+const parseFromHls = (main: string, playlists: string[]) => {
+  const hamObj = hlsToHam(main, playlists); // Convert HLS to HAM object
+  const hlsManifest = hamToHls(hamObj); // Convert HAM object to HLS manifest
+  const dashManifest = hamToDash(hamObj); // Convert HAM object to DASH manifest
+};
```
Comment (DuckyCB) [inline lib/src/cmaf/README.md]: Same as previous comment, remove the arrow function (it's weird) and define the variables with complete names.

Probably creating a new code block to define the hls functions, so the name of the variables can be repeated

Comment (NoeliaBentancor) [reply lib/src/cmaf/README.md]: I replaced arrow functions, let me know if it makes sense for you the way it is right now

**Review by matiasrb97 — APPROVED** [review]

**Review by DuckyCB — APPROVED** [review]

**Pull request merged**